src/d3a/models/market/two_sided_pay_as_clear.py

```diff
@@ -74,7 +76,10 @@ def _get_clearing_point(self, max_rate):
         for i in range(1, max_rate + 1):
             if self.state.cumulative_offers[self.now][i] >= \
                     self.state.cumulative_bids[self.now][i]:
-                return i, self.state.cumulative_bids[self.now][i]
+                if self.state.cumulative_bids[self.now][i] == 0:
+                    return i-1, self.state.cumulative_offers[self.now][i-1]
+                else:
+                    return i, self.state.cumulative_bids[self.now][i]
```

Comment:
I think i need an offline explanation for this part.

Reply:
Sure, ping me when you are free

Reply:
Discussed it offlline. Makes sense now. Please rename i to something like `rate`, Thanks!